python/cugraph/cugraph/dask/community/louvain.py

```diff
@@ -45,19 +45,29 @@ def convert_to_cudf(result: cp.ndarray) -> Tuple[cudf.DataFrame, float]:
 
 
 def _call_plc_louvain(
-    sID: bytes, mg_graph_x, max_iter: int, resolution: int, do_expensive_check: bool
+    sID: bytes,
+    mg_graph_x,
+    max_iter: int,
+    threshold: float,
+    resolution: float,
```

Comment:
Resolution was called type int.  Changed it to float, as that seems incorrect.  Or is that an undocumented trick to get around the fact that double wasn't supported?